rules/cloud/azure/audit_logs/azure_aad_secops_ca_policy_updatedby_bad_actor.yml

```diff
@@ -15,9 +15,9 @@ logsource:
     product: azure
     service: auditlogs
 detection:
-    keywords:
-        - Update conditional access policy
-    condition: keywords
+    selection:
+        properties.message: Update conditional access policy
+    condition: selection
```

Comment:
Hello,
thanks for the update.
As the detection change the field `modified` must be add below the date.
like :
```yaml
date: 2022/07/19
modified: 2024/05/28
```

Reply:
Sorry I missed this. That's now been added.